manifests/params.pp

```diff
@@ -29,9 +29,21 @@
   if $::operatingsystem == 'Ubuntu' {
     if versioncmp($::lsbdistrelease, '8.04') < 1 {
       $init_style = 'debian'
-    } else {
+    } elsif versioncmp($::operatingsystemrelease, '15.04') < 0 {
```

Comment:
I wasn't aware. Then lets change this to `$::lsbdistrelease` as well since we know it exists even on the latest versions. My concern is we don't want to troubleshoot a bug where these two facts have different values.